pkg/helpers/helpers.go

```diff
@@ -211,6 +211,8 @@ func GetCloudTargetEnv(location string) string {
 		return "AzureGermanCloud"
 	case strings.HasPrefix(loc, "usgov") || strings.HasPrefix(loc, "usdod"):
 		return "AzureUSGovernmentCloud"
+	case strings.HasPrefix(strings.ToLower(loc), "usnat") || strings.HasPrefix(strings.ToLower(loc), "ussec"):
```

Comment:
cc @xizhamsft

Reply:
I will prepare the PR for aks and ping @xizhamsft

Reply:
No please don't have this here.

Reply:
Removed. I thought it does not matter. you can find the location in the list https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.documents.locationnames?view=azure-dotnet